backend/manifests/vars/vars-dev.yml

```diff
@@ -0,0 +1,21 @@
+# gsa-fac manifest
+app_name: gsa-fac
+mem_amount: 512M
+cf_env_name: DEVELOPMENT
+env_name: dev
+service_name: dev
```

Comment:
These three variables feel both confusing and redundant... Their values are nearly identical per-environment, and the name of the variable doesn't really help us understand how they're used.

It's not a blocker for this PR, but we should think about how we can further standardize so that the `env_name` is all that's needed across app names, routes, and env vars.

Reply:
cf_env_name will always be caps, and the service_name is used for long, lowercase names.

There is a gap in the original manifests, where for prod, the following `PRODUCTION` `prod` and `production` are used. and then in dev, it is `DEVELOPMENT` and `dev`

the values coming in from inputs.environment are `dev, staging, production`